src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import { TodoFilter } from '../../types/TodoFilter';
+
+type Props = {
+  filter: TodoFilter,
+  setFilter: (newFilter: TodoFilter) => void,
+  activeTodosCount: number,
+  completedTodosCount: number,
+  setIsClearCompleted: (isClearCompleted: boolean) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  filter,
+  setFilter,
+  activeTodosCount,
+  completedTodosCount,
+  setIsClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodosCount === 1
+          ? `${activeTodosCount} item left`
+          : `${activeTodosCount} items left`}
```

Comment:
You only need a check for one word:
```suggestion
        {`${activeTodosCount} ${activeTodosCount === 1 ? 'item' : 'items'} left`}
```